buy_together_app/views.py

```diff
@@ -7,3 +9,34 @@ def main_page(request):
 
 def description_page(request):
     return render(request, 'buy_together_app/description.html')
+
+
+def signup_user(request, msg=""):
+    form = SupplierForm()
+    formm = ClientForm()
+    context = {'form': form, 'formm': formm, 'message': [msg]}
+    return render(request, 'buy_together_app/sign_up.html', context)
+
+
+def signup_supplier(request):
+    if request.method == 'POST':
```

Comment:
You're not testing the else here. What's the expected behavior if this is not POST?

Reply:
Its not supposed to be other than post but Fixed!